supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/Run.scala

```diff
@@ -44,19 +45,20 @@ object Run {
     val logger = new JobLogger("JesRun", jobDescriptor.workflowDescriptor.id, jobDescriptor.key.tag, None, Set(slf4jLogger))
 
     lazy val workflow = jobDescriptor.workflowDescriptor
-    val runtimeInfoBuilder = if (preemptible) PreemptibleJesRuntimeInfoBuilder else NonPreemptibleJesRuntimeInfoBuilder
-    val runtimeInfo = runtimeInfoBuilder.build(commandLine, runtimeAttributes)
+    val pipelineInfoBuilder = if (preemptible) PreemptibleJesPipelineInfoBuilder else NonPreemptibleJesPipelineInfoBuilder
+    val pipelineInfo = pipelineInfoBuilder.build(commandLine, runtimeAttributes)
 
     val pipeline = new Pipeline()
              .setProjectId(projectId)
-             .setDocker(runtimeInfo.docker)
-             .setResources(runtimeInfo.resources)
+             .setDocker(pipelineInfo.docker)
+             .setResources(pipelineInfo.resources)
              .setName(workflow.workflowNamespace.workflow.unqualifiedName)
              .setInputParameters(jesParameters.collect({ case i: JesInput => i.toGooglePipelineParameter }).toVector.asJava)
              .setOutputParameters(jesParameters.collect({ case i: JesFileOutput => i.toGooglePipelineParameter }).toVector.asJava)
 
     def runPipeline: String = {
-      val rpargs = new RunPipelineArgs().setProjectId(projectId).setServiceAccount(JesServiceAccount)
+      val runtimeResources = new PipelineResources().set(noAddressFieldName, runtimeAttributes.noAddress)
```

Comment:
couldn't you take the noAddress out of the pipelineInfo? 

And iff you can, then the string would only be in one place so no need to pull it into a constant (NoAddressFieldName)